pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display summary only for per-process:
+
+`nettop -P`
+
+- Monitor in logging mode and display 10 samples:
+
+`nettop -l {{10}}`
+
+- Monitor in delta mode, updating every 5 seconds:
```

Comment:
```suggestion
- Monitor in delta mode and update every 5 seconds:
```
Follow the standard above you used

Reply:
I wouldn't say that's necessary. A bit diversity in descriptions makes the more interesting.

Reply:
No its not, but it makes the page a little more uniformed